mobile.css

```diff
@@ -0,0 +1,96 @@
+.works {
+  display: grid;
+  width: 330px;
```

Comment:
- [ ] You should not use fixed sized width for it makes your page not responsive. You can use chrome developer tools to see the mobile version while developing.